wellcomeml/ml/doc2vec_vectorizer.py

```diff
@@ -27,6 +28,7 @@ def __init__(self, vector_size=100, window_size=5, n_jobs=1,
             model: underlying model architecture, one of dm or dbow. default: dm
             epochs: number of passes over training data. default: 20
             n_jobs: number of cores to use (-1 for all). default: 1
+            pretrained: pretrained doc2vec model
```

Comment:
I'm not sure its obvious what is wanted here, so maybe specify this is the directory to the model, and also not the directory to the actual doc2vec file (since in _get_model_path you do `"{}/doc2vec".format(model_dir)`)

Reply:
very good point, i was expecting this comment :) i am doing this because more than one files are produced when saving. this parameter is sort of similar to `from_pretrained` that hugging face exposes from loading pretrained BERT like models. i agree though that the name is not super clear. do you have a suggestion? or you mean you happy with a better description in the param docstring?

Reply:
Yeh I think just updating the docstring will be enough